ui/pages/settings/developer-options-tab/developer-options-tab.tsx

```diff
@@ -241,11 +242,54 @@ const DeveloperOptionsTab = () => {
     );
   };
 
+  const renderRemoteFeatureFlags = () => {
```

Comment:
7. Show the `remoteFeatureFlags` value passed to UI. If there's any value in `manifest.json` locally, it will return that value. 
<img width="1026" alt="Screenshot 2025-01-14 at 00 48 22" src="https://github.com/user-attachments/assets/067a4924-74c0-437d-bbf2-45aadf8bc6f8" />